### PR TITLE
Fixes compiler errors in the preload_rsc external resource thingy

### PR DESCRIPTION
**Pull request by MrsTonedOne**
/vg/ found a way to make this work, so i'll be using this system.

Also defaults preload_rsc `0` for test merge reasons. I'll remove that commit once @cyberboss finishes up the define system, and instead switch it to only define if its not already defined.

### CODE REVIEW

**Comment by Cyberboss** [comment]
I have an implementation for the compile time defines in TGS but it won't work with this because it'll get overwritten by that define. Maybe add an #ifndef PRELOAD_RSC around it?

**Comment by MrStonedOne** [comment]
>and instead switch the to only define if its not already defined.

thats already planned

**Comment by Cyberboss** [comment]
TGS now supports including additional files where you can set the define https://github.com/tgstation/tgstation-server/releases/tag/tgstation-server-v3.2.1.10

File: code/game/world.dm [hunk 1]
```diff
@@ -44,13 +44,13 @@ GLOBAL_PROTECT(security_mode)
 
 /world/proc/SetupExternalRSC()
 #if (PRELOAD_RSC == 0)
-	external_rsc_urls = world.file2list("config/external_rsc_urls.txt","\n")
+	GLOB.external_rsc_urls = world.file2list("config/external_rsc_urls.txt","\n")
```
Comment (Cyberboss) [inline code/game/world.dm]: [GLOB.config_dir]/external_rsc_utils.txt

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
Actually nevermind I'll fix it later

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**